sdk/identity/azure-identity/test/ut/token_credential_impl_test.cpp

```diff
@@ -31,30 +31,35 @@ class TokenCredentialImplTester : public TokenCredential {
   HttpMethod m_httpMethod = HttpMethod(std::string());
   Url m_url;
   std::unique_ptr<TokenCredentialImpl> m_tokenCredentialImpl;
+  bool m_proactiveRenewal;
 
 public:
   explicit TokenCredentialImplTester(
       HttpMethod httpMethod,
       Url url,
-      TokenCredentialOptions const& options)
+      TokenCredentialOptions const& options,
+      bool proactiveRenewal = false)
       : TokenCredential("TokenCredentialImplTester"), m_httpMethod(std::move(httpMethod)),
-        m_url(std::move(url)), m_tokenCredentialImpl(new TokenCredentialImpl(options))
+        m_url(std::move(url)), m_tokenCredentialImpl(new TokenCredentialImpl(options)),
+        m_proactiveRenewal(proactiveRenewal)
   {
   }
 
   explicit TokenCredentialImplTester(
       std::function<void()> throwingFunction,
-      TokenCredentialOptions const& options)
+      TokenCredentialOptions const& options,
+      bool proactiveRenewal = false)
```

Comment:
I have to say that I *hate* the term "proactiveRenewal" (because I don't think it correctly describes the feature), but I understand that it is the term that has been used in other SDKs for the same feature.

Reply:
Do you have a proposal for another name?